config.toml

```diff
@@ -88,21 +88,15 @@ breadcrumb_disable = false
 sidebar_search_disable = false
 
 [[params.versions]]
-  version = "v0.11 (latest)"
-  url = "https://release-v0-11.docs.openservicemesh.io/"
+  version = "v1.1 (latest)"
+  url = "https://release-v1-1.docs.openservicemesh.io/"
 [[params.versions]]
-  version = "v0.10"
-  url = "https://release-v0-10.docs.openservicemesh.io/"
-[[params.versions]]
-  version = "v0.9"
-  url = "https://release-v0-9.docs.openservicemesh.io/"
-[[params.versions]]
-  version = "v0.8"
-  url = "https://release-v0-8.docs.openservicemesh.io/"
+  version = "v1.0"
+  url = "https://release-v1-0.docs.openservicemesh.io/"
 [params.versionbanner]
 	show = true
 	archive = "v0.8"
```

Comment:
just wondering, why does this default to 0.8? 
<img width="497" alt="image" src="https://user-images.githubusercontent.com/42152676/175648717-11022fec-6c1b-4b73-affb-ebec9fbc9970.png">

Reply:
oh, just browsed through more of the preview -- is it just that 0.8 is the lowest version we support?